docusaurus/docs/develop/path/3_cheatsheet_shannon.md

```diff
@@ -207,6 +207,53 @@ hydrator_config:
      --node=https://shannon-testnet-grove-rpc.beta.poktroll.com
    ```
 
+:::
+
+:::info Service QoS Hydrator Checks
```

Comment:
@commoddity If your admonition starts looking like this, you probably just want to make it a new section 

(I'm making the edits and will push shortly)

![Screenshot 2025-05-06 at 1 33 05 PM](https://github.com/user-attachments/assets/d4e203f0-d36c-427e-9929-340c4219694e)